crash-handler-process/metricsprovider.cpp

```diff
@@ -393,6 +398,12 @@ std::string MetricsProvider::GetMetricsFileStatus()
 
 void MetricsProvider::SendMetricsReport(std::string status)
 {
+    // Disabled on dev env
+    if (m_IsDevEnv)
+    {
+        return;
+    }
```

Comment:
seems like the brackets are not needed here

Reply:
Done